modoboa_installer/utils.py

```diff
@@ -75,11 +75,12 @@ def dist_name():
 
 def mkdir(path, mode, uid, gid):
     """Create a directory."""
-    if not os.path.exists(path):
-        os.mkdir(path, mode)
+    dir = os.path.dirname(path)
```

Comment:
Why are you doing that? Is there a place in the installer where I call this function with a file path instead of a directory one?